Source/Resources/Content/MSGraph/ConditionalAccess/Policies/Require MFA from Unmanaged Devices and Untrusted Locations.json

```diff
@@ -44,7 +44,7 @@
       ]
     }
   },
-  "description": "Requires MFA when authenticating off-network.",
+  "description": "What is it: Requires MFA when authenticating from a non-compliant device from off-network. \n Simeon Notes: Protects your data by ensuring a device connecting to your network is compliant. Simeon's baseline includes all applications and users except those users with the Azure AD role 'Directory Synchronization Accounts'. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/untrusted-networks",
```

Comment:
**What does this do?**
Requires MFA when authenticating from a non-compliant device that is off-network. The baseline includes all applications and users except those users with the Azure AD role 'Directory Synchronization Accounts'. 

**Why should you use this?**
This further protects your data by ensuring that a device connecting to your network is compliant. 

**What is the end-user impact?**
Users must authenticate using MFA when logging in from a non-compliant, off-network device. <-- @metamenos is this correct to say?

**Learn more**
[Protect your Microsoft 365 global administrator accounts](How to: Require MFA for access from untrusted networks with Conditional Access)